backend/pom.xml

```diff
@@ -84,6 +84,11 @@
 			<artifactId>javax.ws.rs-api</artifactId>
 			<version>2.1.1</version>
 		</dependency>
+		<dependency>
+			<groupId>com.google.firebase</groupId>
+			<artifactId>firebase-admin</artifactId>
+			<version>7.1.0</version>
+		</dependency>
```

Comment:
Just for fun?

Reply:
Próbálgattuk és bent felejtettük.
Természetesen kikerül